src/control/lib/daos/api/api_test.go

```diff
@@ -0,0 +1,28 @@
+//
+// (C) Copyright 2025 Google LLC
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package api
+
+import (
+	"os"
+	"testing"
+)
+
+// TestMain defines package-level setup and teardown logic.
+// NB: Any packages that run tests which depend on the API stubs
+// should copy this function in order to avoid interference
+// between parallel tests. Go parallelizes testing across
+// packages (but not within a package) by default.
+//
+// Long-term, this should be phased out as API users should mock
+// the API instead of relying on the stubs.
```

Comment:
IMO we probably need a task to change any existing code at the higher level so people don't imitate it.

Reply:
Yeah, good point. I see these as temporary scaffolding for the construction of the API. I'll update the comment to make that more explicit.